scripts/globals/events/harvest_festivals.lua

```diff
@@ -5,6 +5,231 @@ require("scripts/globals/settings")
+                111.054, -2.000, 84.144,
+                123.245, -2.112, 94.284,
+                125.542, -2.500, 113.878,
+                148.683, -2.500, 136.279,
+                148.662, 0.000, 183.685,
+                118.668, 0.000, 184.553,
+                116.580, 0.000, 169.693,
+                98.244, -2.500, 171.419,
+                98.426, -2.500, 202.653,
+                145, 0, 211
+            },
+            {
+                8.888, -1.000, 22.891,
+                8.788, -1.000, 22.691,
+                -7.035, -1.000, 21.795,
+                -8.821, -2.000, -24.073,
+                -45.499, -2.632, -65.225,
+                -43.493, -3.401, -75.936,
+                -36.120, -2.500, -90.229,
+                -36.084, -2.500, -102.374,
+                -4.649, -2.500, -135.233,
+                -4.321, -4.000, -181.148,
+                -42.469, -1.000, -180.298,
+                -76.639, -1.000, -163.456,
+                -112.138, -1.000, -162.446,
+                -155.553, -2.166, -180.748,
+                -176.367, -1.492, -180.721,
+                -189.488, -0.054, -170.694,
+                -203.669, -1.640, -173.720,
+                -205.304, -1.808, -184.690,
+                -225.577, -1.907, -224.560,
+                -242.360, -3.104, -231.883,
+                -245.484, -3.000, -254.458,
+                -242.231, -3.060, -268.685,
+                -243.057, -3.000, -295.611,
+                -236.585, -3.000, -295.444,
+                -239.779, -2.766, -266.985,
+                -220.064, -1.610, -224.771,
+                -218.331, -0.475, -212.793,
+                -201.731, -1.943, -179.814,
+                -222.445, -1.274, -139.862,
+                -220.428, -3.132, -120.915,
+                -205.092, -3.250, -111.478,
+                -210.662, -2.302, -134.882,
+                -219.147, -0.633, -145.515,
+                -205.863, -1.681, -172.294,
+                -185.662, -0.296, -173.420,
+                -142.489, -1.805, -179.430,
+                -108.894, -1.000, -159.810,
+                -84.731, -1.000, -160.901,
+                -53.749, -1.000, -169.109,
+                -50.925, -2.500, -118.561,
+                -35.840, -2.500, -100.982,
+                -36.630, -2.500, -86.128,
+                -40.215, -3.000, -56.350,
+                -7.831, -2.000, -23.366,
+                -9.264, -1.000, 31.940,
+                9.922, -1.000, 34.360,
+                9, 0, 23
+            },
+        },
+    },
+}
+
+function events.harvest.initZone(zone)
+    if isHalloweenEnabled() ~= 0 then
+        applyHalloweenNpcCostumes(zone:getID(),true)
+        applyHalloweenDecorations(zone:getID(),true)
+    end
+    events.harvest.registerRegions(zone)
+end
+
+function events.harvest.registerRegions(zone)
+    switch (zone:getID()): caseof
+    {
+        [230] = zone:registerRegion( 7,  122.602,  0.0,   72.500,  125.000,  0.0,   75.625 ), --Southern_San_dOria Pitchfork
```

Comment:
inconsistency here trips my OCD. functions contents isn't indented, the comments aren't up to standard `-- space` instead of `--nospace` and while it may not technically be a problem having some be function contents and others not really throws me off reading it.

I'll let someone else weigh in because unfortunately my er..condition..causes me to zero in on formatting and miss the bigger picture, which isn't fair to you.

Reply:
It's all fair I'll fix up all that you mentioned when I get home tonight